docs/concepts/gas-enabled-network.md

```diff
@@ -0,0 +1,83 @@
+---
+description: Gas enabled networks
+---
+
+# Gas enabled networks
+
+By default, GoQuorum is a [free gas network](free-gas-network.md), which means gas is not consumed by transactions.
```

Comment:
which means gas is priced at zero and is not consumed when transactions are processed.